packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -132,6 +132,13 @@ export interface BrowserTracingOptions {
    */
   traceXHR: boolean;
 
+  /**
+   * Flag to disable tracing of long-lived streams, like server-sent events (SSE) via fetch.
+   *
+   * Default: false
+   */
+  traceStreams: boolean;
```

Comment:
Bikeshedding: I'd vote for naming this option something like `trackFetchStreamPerformance` (maybe there's something even better). The reason is that what we call "tracing" happens regardless of this option.